src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -954,5 +954,58 @@ string GetPropertyValue(string propertyName)
 
             GetPropertyValue("RootNamespace").Should().Be("Project_Name_With_Spaces");
         }
+
+        [Fact]
+        public void It_errors_on_windows_sdk_assembly_version_conflicts()
+        {
+            var testProjectA = new TestProject()
+            {
+                Name = "ProjA",
+                TargetFrameworks = "net5.0-windows10.0.19041"
+            };
+            testProjectA.SourceFiles.Add("ProjA.cs", @"namespace ProjA
+{
+    public class ProjAClass
+    {
+        public static string str { get { return Windows.Media.Devices.ColorTemperaturePreset.Auto.ToString(); } }
+        public string ProjBstr { get { return ProjB.ProjBClass.str; } }
+    }
+}");
+            var testProjectB = new TestProject()
+            {
+                Name = "ProjB",
+                TargetFrameworks = "net5.0-windows10.0.19041",
+            };
+            testProjectB.SourceFiles.Add("ProjB.cs", @"namespace ProjB
+{
+    public class ProjBClass
+    {
+        public static string str { get { return Windows.Media.Devices.ColorTemperaturePreset.Auto.ToString(); } }
+    }
+}");
+            testProjectA.ReferencedProjects.Add(testProjectB);
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProjectA)
+                .WithProjectChanges((path, project) =>
+                {
+                    if (Path.GetFileName(path).Equals("ProjA.csproj"))
+                    {
+                        var ns = project.Root.Name.Namespace;
+                        var itemGroup = project.Root.Elements(ns + "ItemGroup").First();
+                        var updateKnownFrameworkRef = new XElement(ns + "KnownFrameworkReference",
+                            new XAttribute("Update", "Microsoft.Windows.SDK.NET.Ref"),
+                            new XAttribute("TargetingPackVersion", "10.0.18362.12"));
```

Comment:
We still want the targeting pack to match the version of Windows we're targeting, we just want a previous patch version of the targeting pack.

```suggestion
                            new XAttribute("TargetingPackVersion", "10.0.19041.10"));
```

Reply:
Making this change causes the test to not repro the issue

Reply:
Looks like they are not updating the assembly versions in newer packages in order to avoid this issue.  They will update it when they add new APIs.

Try using version 10.0.19041.6-preview here and see if that repros the issue.

Reply:
Yup, that repros :)